README.md

```diff
@@ -303,6 +303,7 @@ defaults):
 | `spel_epel7release`     | URL to the release RPM for the [EPEL 7][10] repo                  |
 | `spel_epelrepo`         | Name of the epel repo (if different than "epel")                  |
 | `spel_extrarpms`        | Comma-separated list of extra package/@group names to pass to yum |
+| `spel_devnode`          | Device-path for the boot disk: override if value in standard builder-definition is not valid for build-environment (e.g., if building on AWS Nitro instance-types, set to `/dev/nvme0n1` |
```

Comment:
Maybe tone down the verbosity a touch... this makes the table unbearably large...

```suggestion
| `spel_devnode`          | Device-path for the boot disk, e.g. `/dev/nvme0n1` for AWS Nitro |
```

![image](https://user-images.githubusercontent.com/8457307/76625098-cc8f5f00-64f3-11ea-965d-1c174d0fce23.png)

Reply:
Weird... Just linewraps on mine:

![image](https://user-images.githubusercontent.com/7087031/76625643-184f0200-650e-11ea-8e8f-e1c62e161385.png)

Any way, I'll adjust it ...moar reasons I hate "simplified" markup languages.

Reply:
it's this enormous monitor i've got 😂

Reply:
If a monitor requires me to move my head to read across a line of text, the monitor is too big. :stuck_out_tongue: